### ISSUE TITLE
Error: "No signature of method..." when following auto-update route 0.9.7-->0.10.0

### ISSUE THREAD

**Issue by iainism** [jmfayard/refreshVersions#365]
- [x] I have read the [guidelines for submitting issues](https://jmfayard.github.io/refreshVersions/contributing/submitting-issues/)
## 🐛 Describe the bug
I just ran `gradlew refreshVersions` to be told by it that an update is available, and to see `settings.gradle`.  I went and edited the file, and now I get an error when trying to run refreshVersions again.

In settings.gradle I now have:
```
import de.fayard.refreshVersions.RefreshVersionsSetup

import de.fayard.refreshVersions.RefreshVersionsMigration

buildscript {
    repositories { gradlePluginPortal() }
    dependencies.classpath("de.fayard.refreshVersions:refreshVersions:0.10.0")
}

plugins {
    id "com.gradle.enterprise" version "3.2"
}

//gradleEnterprise {
//    buildScan {
//        termsOfServiceUrl = "https://gradle.com/terms-of-service"
//        termsOfServiceAgree = "yes"
//        allowUntrustedServer = true
//    }
//}

RefreshVersionsMigration.migrateIfNeeded("0.9.7") // Will be automatically removed by refreshVersions when upgraded to the latest version.

RefreshVersionsSetup.bootstrap(settings)
```

## ⚠️ Current behavior
```
Settings file 'C:\Users\vubcia\AndroidStudioProjects\...\settings.gradle' line: 22

A problem occurred evaluating settings '...'.
> No signature of method: static de.fayard.refreshVersions.RefreshVersionsMigration.migrateIfNeeded() is applicable for argument types: (String) values: [0.9.7]
  Possible solutions: migrateIfNeeded(org.gradle.api.initialization.Settings, java.lang.String)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

```
## ✅ Expected behavior
Refresh Versions runs successfully

## 💣 Steps to reproduce
Update from 0.9.7

## 📱 Tech info
Solution: Add `getSettings()` to migrateIfNeeded() call: `RefreshVersionsMigration.migrateIfNeeded(**getSettings()**, "0.9.7") // Will be automatically removed by refreshVersions when upgraded to the latest version.`

**Comment by LouisCAD** [comment]
Hello, thanks for your report, I'll fix it for the next release so you can upgrade straight from 0.9.7, skipping 0.10.0

**Comment by LouisCAD** [comment]
Hello, sorry for the trouble. Actually, it's unfortunately too late for us to be able to fix the issue (no longer relevant in 0.10.0+).

However, it should not block you so long you ditch the `migrateIfNeeded` call and upgrade to the version `0.10.0` directly.
